minifiers/config.go

```diff
@@ -27,54 +27,64 @@ import (
 	"github.com/tdewolff/minify/v2/xml"
 )
 
-var DefaultConfig = minifiersConfig{
-	EnableHtml: true,
-	EnableCss:  true,
-	EnableJs:   true,
-	EnableJson: true,
-	EnableSvg:  true,
-	EnableXml:  true,
-
-	Html: html.Minifier{
+var defaultTdewolffConfig = tdewolffConfig{
+	EnableHTML: true,
```

Comment:
I think we can assume that any future minifier will support all of these fomats, so I suggest we keep all the Enable* on top level.

Reply:
done!
thanks you for reviewing